src/azure-cli/azure/cli/command_modules/dms/_help.py

```diff
@@ -61,7 +61,7 @@
   - name: --delete-running-tasks
     type: bool
     short-summary: >
-        Cancel any running tasks before deleting the service.
+        Cancel any running Tasks before deleting the Service.
```

Comment:
prefer previous help message. `Task` should be used at the beginning of the sentence.
The same for above changes